lib/v2/components/circular_progress_item.dart

```diff
@@ -28,43 +28,24 @@ class CircularProgressItem extends StatelessWidget {
   Widget build(BuildContext context) {
     return Column(
       children: [
-        Row(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            CircularStepProgressIndicator(
-              circularDirection: CircularDirection.clockwise,
-              totalSteps: totalStep,
-              currentStep: currentStep,
-              stepSize: 2.5,
-              selectedColor: AppColors.green1,
-              unselectedColor: AppColors.darkGreen2,
-              padding: 0,
-              width: circleRadius * 2,
-              height: circleRadius * 2,
-              selectedStepSize: 2.5,
-              roundedCap: (_, __) => true,
-              child: Center(child: icon),
-            ),
-          ],
+        CircularStepProgressIndicator(
+          circularDirection: CircularDirection.clockwise,
+          totalSteps: totalStep,
+          currentStep: currentStep,
+          stepSize: 2.5,
+          selectedColor: AppColors.green1,
+          unselectedColor: AppColors.darkGreen2,
+          padding: 0,
+          width: circleRadius * 2,
+          height: circleRadius * 2,
+          selectedStepSize: 2.5,
+          roundedCap: (_, __) => true,
+          child: Center(child: icon),
         ),
         const SizedBox(height: 8.0),
-        Row(
-          mainAxisAlignment: MainAxisAlignment.center,
-          mainAxisSize: MainAxisSize.min,
-          children: [
-            Flexible(
-              child: ConstrainedBox(
-                constraints: const BoxConstraints(maxWidth: 120),
-                child: Text(title, textAlign: TextAlign.center, maxLines: 2, style: titleStyle),
-              ),
-            ),
```

Comment:
Now I remember the flexible one, in case the username is longer, I don't know how much the limit really is, but only then the long text would cause an overflow if I'm not wrong.